huddle/src/main/java/team/uptech/huddle/HuddleHelper.kt

```diff
@@ -166,6 +168,20 @@ class HuddleHelper(private val view: Huddle, private val parameters: Parameters)
     view.context.getColorIfNotDefault(colorRes) { view.setTextColor(it) }
   }
 
+  private fun Parameters.setupFonts() {
+    setupFont(view.dialogTitle, fonts.titleFont)
+    setupFont(view.dialogMessage, fonts.messageFont)
+    setupFont(view.actionPositive, fonts.ctaFont)
+    setupFont(view.actionNegative, fonts.ctaFont)
+  }
+
+  private fun setupFont(view: TextView, fontRes: Int) {
+    if (fontRes != DEFAULT_FONT) {
+      val typeface = view.context.getFont(fontRes)
+      view.typeface = typeface
```

Comment:
You can remove `val typeface` for me it seems fine like in the code below, but it's up to you. 
```kotlin
private fun setupFont(view: TextView, fontRes: Int) {
  if (fontRes != DEFAULT_FONT) view.typeface = view.context.getFont(fontRes)
}
```